### DISCUSSION TITLE
how to use --root-mode?

### DISCUSSION THREAD

**Discussion by axyz** [swc-project/swc#1784] [Q&A]
Hi, I'm trying to migrate from babel to swc in a monorepo using lerna with yarn workspaces. I don't want to duplicate the `.swrc` files in every nested package and I'm trying to use the one from the root folder. I do not want to pass relative paths as the level of nesting may be variable. With babel I can use `--root-mode upward`  and all the config files will be searched in the root. This feature seem to be available and documented in swc https://github.com/swc-project/swc/blob/master/node-swc/src/types.ts#L90, but passing the option on the command line results in `error: unknown option '--root-mode'` , what is the correct way to provide it?

**Comment by axyz** [comment]
it seems that `-C rootMode=upward` is setting the option correctly, but the config from the root monorepo folder is still no respected, any idea? (I have no other `.swcrc` in any other folder)

**Reply by kdy1** [reply]
I think root mode is not implemented properly. Can you file an issue?

**Reply by axyz** [reply]
sure: https://github.com/swc-project/swc/issues/1801
